samples/bluetooth/broadcaster_multiple/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_BT=y
+CONFIG_BT_BROADCASTER=y
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_EXT_ADV_MAX_ADV_SET=2
+CONFIG_BT_DEVICE_NAME="Broadcaster Multiple"
+
```

Comment:
Should we set `CONFIG_BT_CTLR_ADV_DATA_LEN_MAX` in the prj.conf?

Reply:
I assume that this comment is no longer valid as we have `CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=192` in this config file now